xml/System.Security.Cryptography.X509Certificates/X509ChainPolicy.xml

```diff
@@ -432,17 +432,19 @@
         <ReturnType>System.DateTime</ReturnType>
```

Comment:
`UrlRetrievalTimeout` (~line 353) should say it is the maximum amount of time to be spent during online revocation verification or downloading the certificate revocation list (CRL).  TimeSpan.Zero means "no limit"

Reply:
I'm not seeing a change to UrlRetrievalTimeout in the current set of changes

Reply:
Somehow I missed your comment here. Sorry about that! You should be able to see it now @bartonjs.